Subject: pppd: add crypt options if crypt is switched on
From: Robert Schwebel <r.schwebel@pengutronix.de>

Activate the crypt options (CFLAGS, LIBS) if USE_CRYPT is selected. The
current method is a bad idea, because it relies on the crypt.h header
being installed in a special place, which might not be the case.

The proper solution would probably be a real autotoolization of the
packet, with checks for availability in configure.ac.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 pppd/Makefile.linux |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pppd/Makefile.linux
===================================================================
--- pppd/Makefile.linux.orig
+++ pppd/Makefile.linux
@@ -117,7 +117,7 @@ CFLAGS   += -DHAS_SHADOW
 #LIBS     += -lshadow $(LIBS)
 endif
 
-ifneq ($(wildcard $(SYSROOT)/usr/include/crypt.h),)
+ifdef USE_CRYPT
 CFLAGS  += -DHAVE_CRYPT_H=1
 LIBS	+= -lcrypt
 endif
